static/js/app.ts

```diff
@@ -1435,7 +1435,7 @@ export function toggle_keyword_language(current_lang: string, new_lang: string)
       window.open(hash, "_self");
 
       // if in iframe, reload the topper window level.
-      if (window.top && !(window as any).Cypress) {
+      if (window.top && window.top !== window && !(window as any).Cypress) {
```

Comment:
Nice! Thank you so much for this fix, Brian! This is awesome stuff, we appreciate your contributions a lot, sorry I havent been able to review them sooner.